test/DebuggerCommon/ES6_proto_invalidation.js.dbg.baseline

```diff
@@ -740,10 +739,9 @@
             },
             "indexOf": {
               "#__proto__": "function <large string>",
-              "length": "number 1",
-              "name": "string indexOf",
-              "caller": "object null",
-              "arguments": "object null"
+              "prototype": "Object {...}",
+              "name": "string arrayIndexOf",
+              "length": "number 1"

```

Comment:
I think my comment got buried, but this change looks like a change in observable semantics. We want to make sure that that js implementation is not observably different from the native implementation we had before. Some of these behaviors are by spec, like the name being "arrayIndexOf" instead of "indexOf".

In order to manipulate this you might actually need to follow the patterns toward the bottom of Intl.js for adding functions to the various Intl objects.